euclidlib/photo/_le3_2pcf_wl.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from os import PathLike
+from typing import TYPE_CHECKING, Tuple, Union
+from numpy.typing import NDArray
+
+from euclidlib.photo import photo_data
+
+if TYPE_CHECKING:
+    from collections.abc import Sequence
+    from typing import Any, TypeAlias
+
+
+# type alias
+_DictKey: "TypeAlias" = Union[str, int, Tuple["_DictKey", ...]]
+
+def xi_tpcf(
```

Comment:
I think this needs a more expressive name, like how we decided on `angular_power_spectra()` and `mixing_matrices()`. See other comment.

Reply:
I changed it to `two_point_correlation`, since we are planning to read the other FITS files as well. Let me know if you prefer something different.